customer/src/test/java/no/unit/nva/customer/testing/CreateUpdateControlledVocabularySettingsTests.java

```diff
@@ -0,0 +1,110 @@
+package no.unit.nva.customer.testing;
+
+import static no.unit.nva.customer.ControlledVocabularyHandler.IDENTIFIER_PATH_PARAMETER;
+import static no.unit.nva.customer.ObjectMapperConfig.objectMapper;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.mock;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.net.HttpHeaders;
+import com.google.common.net.MediaType;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.UUID;
+import no.unit.nva.customer.ControlledVocabularyHandler;
+import no.unit.nva.customer.get.GetControlledVocabularyHandler;
+import no.unit.nva.customer.model.CustomerDto;
+import no.unit.nva.customer.model.CustomerDtoWithoutContext;
+import no.unit.nva.customer.model.interfaces.VocabularySettingsList;
+import no.unit.nva.customer.service.impl.DynamoDBCustomerService;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.apigateway.MediaTypes;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JsonUtils;
+import org.junit.jupiter.api.BeforeEach;
+
+public abstract class CreateUpdateControlledVocabularySettingsTests extends CustomerDynamoDBLocal {
```

Comment:
Class creating common logic for tests regarding creating and updating vocabulary settings